src/hooks/useDebounce.ts

```diff
@@ -0,0 +1,24 @@
+import { useEffect, useState } from "react";
```

Comment:
this file can be overloaded with https://www.npmjs.com/package/rooks - I suggest this because it can also serve as a quick fix for #205

Reply:
Opened an issue for that https://github.com/open-sauced/hot/issues/228